unity/ComputerVision-UnityClient/Assets/Scripts/DataContracts.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System.Runtime.Serialization.Json;
+using System.Text;
+using System.Runtime.Serialization;
+using System.IO;
+namespace MobiledgeXComputerVision
+{
+    [DataContract]
+    public class FaceDetectionResponse
+    {
+        [DataMember]
+        public bool success;
+        [DataMember]
+        public float server_processing_time;
+        [DataMember]
+        public int[][] rects;
+    }
+
+    [DataContract]
+    public class FaceRecognitionResponse
+    {
+        [DataMember]
+        public bool success;
+        [DataMember]
+        public string subject;
+        [DataMember]
+        public float confidence;
+        [DataMember]
+        public float server_processing_time;
+        [DataMember]
+        public int[] rect;
+    }
+
+    [DataContract]
+    public class PoseDetectionResponse
+    {
+        [DataMember]
+        public bool success;
+        [DataMember]
+        public float server_processing_time;
+        [DataMember]
+        public float[][][] poses;
+    }
+
+    [DataContract]
+    public class ObjectDetectionResponse
+    {
+        [DataMember]
+        public bool success;
+        [DataMember]
+        public float server_processing_time;
+        [DataMember]
+        public bool gpu_support;
+        [DataMember]
+        public @Object[] objects;
+    }
+
+    [DataContract]
+    public class @Object
```

Comment:
What does the @ symbol do?

Reply:
Since Object is reserved keyword so adding @ make it usable